### PR TITLE
JENKINS-72640: use Jenkins LTS version 2.426.3 as base

### PR DESCRIPTION
**Pull request by krulls**
### Testing done

I just tested the usual Maven build with 'mvn clean package' which is running unit tests.

```[tasklist]
### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

relates to https://issues.jenkins.io/browse/JENKINS-72640

### CODE REVIEW

File: pom.xml
Comment (krulls) [inline pom.xml:1]: Do you know why in line 70 the tag hasn't been updated with the last release (of version 1.18)?

**Comment by MarkEWaite** [comment]
Thanks for the pull request @krulls .  The Jenkins documentation ["Choosing a Jenkins version to build against"](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/) recommends either Jenkins 2.401.3 or Jenkins 2.414.3 as minimum version, rather than 2.426.3.  

Was there a specific reason that you chose 2.426.3 instead of choosing one of the versions recommended by the documentation?

If you'd like more information about updating the minimum Jenkins version, there is a [dedicated tutorial section](https://www.jenkins.io/doc/developer/tutorial-improve/update-base-jenkins-version/) available with a [video](https://youtu.be/Fev8KfFsPZE?t=2430).

The [plugin version report](https://stats.jenkins.io/pluginversions/thinBackup.html) shows:

* 32% of release 1.18 installs are on 2.426.2 and newer
* 69% of release 1.18 installs are on 2.414.3 and newer
* 88% of release 1.18 installs are on 2.401.3 and newer 

By choosing 2.401.3 more users will be able to install the next release.  By choosing 2.414.3, there will be more time before the next change of minimum Jenkins version.

**Comment by StefanSpieker** [comment]
Thanks for the pull request and your interest here @krulls 
@MarkEWaite thanks a lot for the detailed explanations regarding the versions.

Since many of the users are still on an older Jenkins versions, I would also like to use **small steps** to **not** exclude many users from the update. The version I choose was mainly because it is the first version being on java 11, but I guess it is time to move forward. 

I would appreciate it if you would adjust it to `2.401.3`, if you do not have a specific reason for `2.426.3`

File: pom.xml
Comment (MarkEWaite) [reply pom.xml:1]: I think that indicates that something failed in the 1.17 release.  The Jenkins plugin archetype inserts a symbolic value there that is replaced by the Apache Maven release plugin.  It usually is:

```xml
<tag>${scmTag}</tag>
```

**Comment by krulls** [comment]
@MarkEWaite : Thank you for pointing out the docs and statistics. I wasn't aware of the documentation and since we use LTS versions for a long time already, I am used to choose them as a base version. Next to that I wanted to see which version of the _Apache commons-io_ library would be used and if it would help to fix https://issues.jenkins.io/browse/JENKINS-72641. But since it doesn't I have no benefit from using the LTS version. And, of course, it makes most sense to use a version that the majority of current thinBackup users are having installed.

**Comment by krulls** [comment]
@StefanSpieker : Have set the preferred base version and fixed this scm tag thingy in the pom. I ran a 'mvn clean package' locally and compilation and unit tests went successful.

**Comment by StefanSpieker** [comment]
Thanks for this PR. I hope I will find some time next weekend to check the bug you are mentioning. This weekend I'm on FOSDEM and will not find the time for digging in.

**Comment by MarkEWaite** [comment]
@StefanSpieker or @krulls could you change the title of this pull request to match the actual change?  If release drafter is being used, that will make the changelog easier to understand.

**Pull request merged**